src/modules/contacts.js

```diff
@@ -0,0 +1,397 @@
+    let opened = this.state.infoOpened[1] ? false : true;
+    this.setState({infoOpened: [id, opened]})
+  }
+
+  // Gets specified attribute from numbers object
+  getAttribure (attribute, id) {
+    let value = "";
+
+    this.state.numbers.forEach((person, index, array) => {
+      if(person.number == id){
+        value = this.state.numbers[index][attribute]
+      }
+    })
+
+    return value
+  }
+
+  render() {
+    // Defining modules
+    let notificationBirthDay = this.state.notification ? 
+      <div className="notification" onClick={() => this.setState({notification: this.state.notification ? false : true})}>{`${this.birthDays[0]} has birthday today! He/she likes ${getLikes(this.state.numbers,this.birthDays[0])}`}</div> : 
+      <div className="notification--closed"></div>;
+
+    let notificationCall = this.state.unCalled ? 
+      <div className="unCalled" onClick={() => this.setState({unCalled: this.state.unCalled ? false : true})}>{`You havent called ${this.unCalled[0]} in a month! Do something about it :)`}</div> : 
+      <div className="notification--closed"></div>;
+
+    let message = this.state.filled ? 
+      <p className="none-message"></p> : 
+      <p className="error-message">Please fill out name and number</p>;
+
+    const AddContactForm = (props) => (
+      <div className="contact__add open">
+        <div className="contact__input__wrapper">
+          <span>Name:</span>
+          <input type="text" className="contact__add-name" />
+        </div>
+        <div className="contact__input__wrapper">
+          <span>Number:</span>
+          <input type="number" className="contact__add-name" />
+        </div>
+        <div className="contact__input__wrapper">
+          <span>Birth day:</span>
+          <select className="contact__add-day">
+            {this.days.map(day => <option value={day}>{day}</option>)}
+          </select>
+          <select className="contact__add-day">
+            {this.months.map(month => <option value={month}>{month}</option>)}
+          </select>
+          <select className="contact__add-day">
+            {this.years.map(year => <option value={year}>{year}</option>)}
+          </select>
+        </div>
+        <div className="contact__input__wrapper">
+          <span className="contact__add-likes-span">Likes:</span>
+          <input type="text" className="contact__add-likes" />
+        </div>
+        {message}
+        <div className="contact__add-button" onClick={this.createContact}>
+          Save
+        </div>
+        <div className="contact__add-button contact__add-button--delete" onClick={this.beingAdded}>
+          Close
+        </div>
+      </div>
+    )
+
+    let Button = this.state.adding ? <AddContactForm click={this.beingAdded}/> : <AddContact click={this.beingAdded} />;
+
+    let infoWindow = this.state.infoOpened[1] ? 
+      <InfoWindow  
+        message={message} 
+        clickOne={this.changeContact}
```

Comment:
Číslování onCLick eventů považuji za antipatern protože název nic neříká o tom, co ta metoda vlastně dělá takže každý člověk co na to má navazovat se musí proklikat až k použití té metody, kouknout se kdy se vlastně volá a co má vlastně dělat.
lepší by bylo něco ve stylu onClickDelete, onCLickInfo atp.

Reply:
Zde souhlasim, ze popisovani click event funkci se dalo pojmenovat lepe.

Reply:
:+1: